### PR TITLE
ci: trim down the workflows and updates docs

### PR DESCRIPTION
**Pull request by akrigline**
- comment out `test` related workflows for now
- change release tags to be `dnd5e` instead of `fvtt`
- update instructions accordingly in readme
- update links in contributing with new milestones

### CODE REVIEW

**Comment by akrigline** [comment]
blocked by #15, which fixes tests. will rebase this on top of dnd5e-1.2.4 when that's up and force push

**Comment by DavidArchibald** [comment]
Go ahead and fix the PR now.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ The following is a set of guidelines for contributing to foundry-vtt-dnd5e-types
 
 - Documentation: [README.md](README.md), [Wiki](https://github.com/League-of-Foundry-Developers/foundry-vtt-dnd5e-types/wiki)
 - Issue tracker: [Issues](https://github.com/League-of-Foundry-Developers/foundry-vtt-dnd5e-types/issues)
-- Project boards
-  - [0.7.9](https://github.com/League-of-Foundry-Developers/foundry-vtt-dnd5e-types/projects/1)
-  - [0.8.0](https://github.com/League-of-Foundry-Developers/foundry-vtt-dnd5e-types/projects/2)
+- Milestones
+  - [1.2.4](https://github.com/League-of-Foundry-Developers/foundry-vtt-dnd5e-types/milestone/1)
+  - [1.3.0](https://github.com/League-of-Foundry-Developers/foundry-vtt-dnd5e-types/milestone//2)
```
Comment (DavidArchibald) [inline CONTRIBUTING.md]: Double slash typo here

File: .github/workflows/checks.yml [hunk 2]
```diff
@@ -49,23 +49,23 @@ jobs:
...
-        with:
-          path: .npm
-          key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.OS }}-node-
-            ${{ runner.OS }}-
-      - run: npm ci --cache .npm --prefer-offline
-      - name: execute tests
-        run: npm test
+  # test:                                     # we have no tests
```
Comment (DavidArchibald) [inline .github/workflows/checks.yml]: Comments might be better completely removed from source

**Review by DavidArchibald — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (akrigline) [reply CONTRIBUTING.md]:
```suggestion
  - [1.3.0](https://github.com/League-of-Foundry-Developers/foundry-vtt-dnd5e-types/milestone/2)
```

File: .github/workflows/checks.yml [hunk 3]
```diff
@@ -49,23 +49,23 @@ jobs:
...
+  #       uses: actions/cache@v2
+  #       with:
+  #         path: .npm
+  #         key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+  #         restore-keys: |
+  #           ${{ runner.OS }}-node-
+  #           ${{ runner.OS }}-
+  #     - run: npm ci --cache .npm --prefer-offline
+  #     - name: execute tests
+  #       run: npm test
```
Comment (akrigline) [inline .github/workflows/checks.yml]:
```suggestion
```

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -27,7 +27,7 @@ jobs:
       - run: npm ci --cache .npm --prefer-offline
       - run: tsc
       - run: npm run lint:ci
-      - run: npm test
+      # - run: npm test               # we have no tests
```
Comment (akrigline) [inline .github/workflows/release.yml]:
```suggestion
```

**Review by DavidArchibald — APPROVED** [review]

**Pull request merged**